### ISSUE TITLE
Support elasticsearch over http protocol

### ISSUE THREAD

**Issue by adriancole** [fabric8io/kubernetes-zipkin#87]
Noticed a [complaint over twitter](https://twitter.com/pat2man/status/829381037054517248) wrt " MySQL kept blocking connections due to error rates."

While not sure what was behind this, [mysql is intended for "quick start"](https://github.com/openzipkin/zipkin#mysql), not necessarily production usage.

The most supported options for production are cassandra and elasticsearch. If elasticsearch, usage over the http protocol is the most popular long term route.

Maybe we should add a recipe for this?

**Comment by iocanel** [comment]
Yeah, that was the plan originally, though I never found time. We definitely need to do that at some point.
